web/src/main/webapp/resources/js/custom.js

```diff
@@ -59,4 +72,47 @@ function sendPost(question) {
         },
         error: function(e) {}
     });
+}
+
+function plotScatterplot(payload) {
```

Comment:
I should have asked about this earlier, but why is everything called scatter plot when it is a line plot?